cmd/nerdctl/image_convert.go

```diff
@@ -199,41 +265,6 @@ func imageConvertAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if nydus {
-		if estargz {
-			return errors.New("option --nydus conflicts with --estargz")
-		}
-		if zstdchunked {
-			return errors.New("option --nydus conflicts with --zstdchunked")
-		}
-		if !oci {
-			logrus.Warnln("option --nydus should be used in conjunction with '--oci', forcibly enabling on oci mediatype for nydus conversion")
```

Comment:
Where has this gone?

Reply:
L188-205 to deal with convert conflict, and L260-262 to show this warning.

Reply:
Thanks, but "forcibly enabling on oci mediatype for nydus conversion" seems no longer printed

Reply:
> Thanks, but "forcibly enabling on oci mediatype for nydus conversion" seems no longer printed

Sorry, it was fixed.